src/lib/convertMark/index.ts

```diff
@@ -0,0 +1,27 @@
+const table:{[key:string]:string} = {
+    '<': '&lt;',
+    '>': '&gt;',
+    '"': '&quot;',
+    '\'': '&apos;',
+    '&': '&amp;',
+    '\r': '&#10;',
+    '\n': '<br>'
+};
+
+const strongEmRegex = /(\*\*\*[^<]+\*\*\*)/g
+const strongRegex = /(\*\*[^<]+\*\*)/g
+const emRegex = /(\*[^<]+\*)/g
+
+export const mdConvert = (text:string) => {
+	
+	return text.replace(/[<>"'\r\n&]/g, (chr:string) => {
+        return table[chr]
+    }).replace(strongEmRegex, (chr:string) => {
+        return `<em><strong>${chr.substring(3, chr.length - 3)}</strong></em>`
+    }).replace(strongRegex, (chr:string) => {
+        return `<strong>${chr.substring(2, chr.length - 2)}</strong>`
+    }).replace(emRegex, (chr:string) => {
+        return `<em>${chr.substring(1, chr.length - 1)}</em>`
+    });
+};
+
```

Comment:
I'm not sure how I feel about rolling a custom Markdown parser instead of just keeping Showdown. We'd have to implement a _lot_ of stuff to get to feature parity. Ideally we'd just make sure Showdown is configured so nothing unsafe can get through, in case that's not already the default.